omada-controller.json

```
@@ -0,0 +1,172 @@
+                    },
+                    "29810": {
+                        "description": "Omada Controller and Omada Discovery Utility discover Omada devices through this port. Suggested default: 29810.",
+                        "host_default": 29810,
+                        "label": "Device discovery",
+                        "protocol": "udp"
+                    },
+                    "29811": {
+                        "description": "Omada Controller and Omada Discovery Utility manage Omada devices running firmware fully adapted to Omada Controller v4*. Suggested default: 29811.",
+                        "host_default": 29811,
+                        "label": "Device managing (v4)",
+                        "protocol": "tcp"                    
+                    },
+                    "29812": {
+                        "description": "Omada Controller and Omada Discovery Utility manage/adopt Omada devices running firmware fully adapted to Omada Controller v4*. Suggested default: 29812.",
+                        "host_default": 29812,
+                        "label": "Device adoption (v4)",
+                        "protocol": "tcp"                    
+                    },
+                    "29813": {
+                        "description": "When upgrading the firmware for the Omada devices running firmware fully adapted to Omada Controller v4*. Suggested default: 29813.",
+                        "host_default": 29813,
+                        "label": "Firmware upgrade (v4)",
+                        "protocol": "tcp"                    
+                    },
+                    "29814": {
+                        "description": "Omada Controller and Omada Discovery Utility manage Omada devices running firmware fully adapted to Omada Controller v5*. Suggested default: 29814.",
+                        "host_default": 29814,
+                        "label": "Device managing (v5)",
+                        "protocol": "tcp"                    
+                    }  
+                },
+                "volumes": {
+                    "/opt/tplink/EAPController/data": {
+                        "description": "Choose a DATA and CONFIG Share for Omada Controller files. Eg: create a Share called omada-controller-data for this purpose alone.",
+                        "label": "Controller DATA + CONFIG Storage"
+                    },
+                    "/opt/tplink/EAPController/logs": {
+                        "description": "Choose a LOGS Share for Omada Controller files. Eg: create a Share called omada-controller-logs for this purpose alone.",
+                        "label": "Controller LOGS Storage"
+                    }  
+                },
+                "environment": {
+                    "PUID": {
+                        "description": "Enter the valid UID 'omada' process user ID. It must have full permissions to all Shares mapped in the previous step. Suggested default: 508.",
```

```suggestion
                        "description": "Enter an existing User ID (UID) to run this Rock-on. They must already have full permissions to Shares used by this Rock-on.",
```
Bit if a re-work/wording from what we have used before but I'm attempting to improve on our clarity here. Plus the 508 UID default does not make sense and it would be less advisable to encourage folks to start making < 1000 UIDs on purpose. By default all new users created have UID >= 1000, and incidentally a GID of 100 (users).